### ISSUE TITLE
Set point values in ACP?

### ISSUE THREAD

**Issue by TRGH2016** [NicolasSiver/nodebb-plugin-ns-points#7]
Just thought that being able to manually enter a value to use as a number of points to award from within the ACP would be awesome. There will be certain things that some admins wish to reward, while others may not.

While the plugin could be edited locally, its a pain to have to note which plugins you have to keep an eye on to re-edit after an upgrade.

**Comment by NicolasSiver** [comment]
Yes, agree. Actually, I have something like this in TODO list (very first element) - https://github.com/NicolasSiver/nodebb-plugin-ns-points#todo
